test/lazyload.test.js

```diff
@@ -0,0 +1,67 @@
+import { lazyloadOptions, loadAllNow } from '..';
+
+lazyloadOptions.nativeLazyloadEnabled = false;
+
+const setup = () => {
+  document.body.innerHTML = `
+    <div style="height: 1500px;"></div>
+    <img src="" srcset="" data-lazy data-srcset="https://picsum.photos/seed/picsum/200/300" alt="">
+    <img src="" data-lazy data-src="https://picsum.photos/seed/picsum/200/300" alt="">
+    <iframe src="" data-lazy data-src="https://firstandthird.com/" title="First+Third Website"></iframe>
+    <video src="" data-lazy data-src="https://www.w3schools.com/html/movie.mp4" controls></video>
+    <audio src="" data-lazy data-src="https://www.w3schools.com/html/horse.mp3"></audio>
+  `;
+
+  loadAllNow();
```

Comment:
I think we should just mock IntersectionObserver since we're not testing that logic at all. https://stackoverflow.com/a/61493088/664071